content/doc/book/security/index.adoc

```diff
@@ -9,30 +9,75 @@ ifdef::backend-html5[]
+The setup wizard enables many of the security options by default,
+to ensure that Jenkins is secure.
+Other security options involve environment-specific setup and trade-offs and depend on specific use cases supported for individual Jenkins instances.
+Configuration options allow you to enable, customize, or disable security features.
+
+[CAUTION]
+====
+Most Jenkins environments grow over time, requiring their trust models to evolve as the environment grows.
+You should schedule regular "check-ups" of your security settings to ensure that they are still appropriate for your instance.
+In particular, if you had to disable some of the default protections that Jenkins provides,
+perhaps because you were using a plugin that had not been updated to support that protection, you may be able to re-enable that protection to increase the security of your instance.
+====
 
 // TODO the following only makes sense on the web site, not the PDF. Can it be disabled there?
+// TODO the material below should be moved to other sections in this chapter.
+
+This chapter introduces the various security options available to Jenkins administrators and users, explaining the protections offered, and trade-offs to disabling some of them:
 
-== Basic Setup
+link:concepts-security[Background Concepts]::
+Discusses security principles that should guide all your decisions about security
+and explains how Jenkins executes a Pipeline so you understand where intrusions can occur if your security configuration is lax.
 
-link:controller-isolation[Controller Isolation]::
+link:controller-isolation[Distributed Builds]::
 Builds should not be executed on the built-in node, but that is just the beginning:
 This section discusses what other steps can be taken to protect the controller from being impacted by running builds. +
 *This needs to be configured according to the needs of your environment.*
 
-link:access-control[Access Control]::
+link:managing-security[Configure Global Security]::
+Much of the basic security configuration is implemented on the _Manage Jenkins >> Configure Global Security_ page.
+Here you see the fields that are configured on that page and get links to other sections that explain each field in detail.
+
+link:access-control[Jenkins Access Control]::
 By default, Jenkins does not allow anonymous access, and a single admin user exists.
 This chapter discusses which level of access is provided by permissions and how to safely grant access to more users. +
 _This is set up securely by the setup wizard. If the setup wizard is disabled on first launch, this may not be configured securely by default._
 
+link:tcp-port[TCP Port for InBound Agents]::
+If you are using TCP for inbound agents (previously called JNLP),
+you must configure a TCP port to use.
+This section discusses how to configure the TCP port and discusses alternative implementations that do not require the TCP port. +
+*The TCP Port is disabled by default and should only be enabled if you need it.*
```

Comment:
This reads weird. As an admin knowing very little about distributed builds, I don't know whether I need this and it looks like I shouldn't use it, but why does it exist?

Perhaps this makes sense as a sub-page of Distributed Builds? Or is there value in keeping this top-level reference around?

Reply:
Agreed -- I just deleted the last sentence.  We say that, if you are using TCP agents for inbound ports, you need to configure the port and this is where you do it.  Enough said, especially for this high-level walk-through.